packages/devtools_app/lib/src/memory/flutter/memory_screen.dart

```diff
@@ -222,29 +191,13 @@ class MemoryBodyState extends State<MemoryBody> {
       children: [
         Row(children: [
           Text(
-            'Memory Source:',
+            'Source:',
             style: TextStyle(fontWeight: FontWeight.bold),
           ),
```

Comment:
I think 'Source' should be inside each item in the dropdown (like it is for the Profile granularity selector).

So your dropdown items would end up being: 
'Source: Live Feed'
'Source: memory_log_xxx'
'Source: memory_log_yyy'

Reply:
Done.